v2/internal/controllers/crd_apimanagement_20220801_test.go

```diff
@@ -111,6 +111,18 @@ func Test_ApiManagement_20220801_CRUD(t *testing.T) {
 				APIM_Api_CRUD(tc, &service)
 			},
 		},
+		testcommon.Subtest{
+			Name: "APIM Product Api CRUD",
+			Test: func(tc *testcommon.KubePerTestContext) {
+				APIM_Product_Api_CRUD(tc, &service)
+			},
+		},
+		testcommon.Subtest{
+			Name: "APIM Product Policy CRUD",
+			Test: func(tc *testcommon.KubePerTestContext) {
+				APIM_Product_Policy_CRUD(tc, &service)
+			},
+		},
```

Comment:
Are these safe to run in parallel with the other tests? The degree of parallelism has me worrying about race conditions.

Reply:
These should be safe to create in parallel as they create different 'named' products to hang the sub-types from.

Reply:
Correct. As long as the service is different we should be good to go